.github/workflows/build_wheels_linux.yml

```diff
@@ -168,20 +173,33 @@ jobs:
           repository: ${{ inputs.repository  }}
           script: ${{ inputs.pre-script }}
       - name: Install python build dependency
```

Comment:
nit: should we refactor it like this ?
```
- name: Build the wheel (python-build-package)
	if: ${{ inputs.build-platform == 'python-build-package' }}
	run: |
	set -euxo pipefail
          # shellcheck disable=SC1090
          source "${BUILD_ENV_FILE}"
          # shellcheck disable=SC2086
          ${CONDA_RUN} python -m pip install build
	 export PYTORCH_VERSION="$(${CONDA_RUN} pip show torch | grep ^Version: | sed 's/Version: *//' | sed 's/+.\+//')"
	${CONDA_RUN} python -m build --wheel

- name: Build the wheel (setup-py)
        if: ${{ inputs.build-platform == 'setup-py' }} 
        working-directory: ${{ inputs.repository }}
        shell: bash -l {0}
        run: |
          set -euxo pipefail
          source "${BUILD_ENV_FILE}"
          ${CONDA_RUN} python setup.py clean
	export PYTORCH_VERSION="$(${CONDA_RUN} pip show torch | grep ^Version: | sed 's/Version: *//' | sed 's/+.\+//')"
	${CONDA_RUN} python setup.py bdist_wheel
```

and remove old ``Build the wheel (bdist_wheel)``